tests/sentry/api/endpoints/test_project_details.py

```diff
@@ -121,13 +121,13 @@ def test_project_renamed_302(self):
         assert (
             AuditLogEntry.objects.get(
                 organization=project.organization, event=AuditLogEntryEvent.PROJECT_EDIT
-            ).data["old_slug"]
+            ).data.get("old_slug")
             == project.slug
         )
         assert (
             AuditLogEntry.objects.get(
                 organization=project.organization, event=AuditLogEntryEvent.PROJECT_EDIT
-            ).data["new_slug"]
+            ).data.get("new_slug")
```

Comment:
These can stay as `data["old_slug"]`, since an assertion failure (in the test) is what you want anyway if the key is not present.